static/js/textMining.js

```diff
@@ -300,8 +299,8 @@ function drawAll(error, dataset) {
     for (var i=0; i<holders.length; i++) {
       var holder = holders[i];
 
-      if (!(holder.cluster == activeCluster) && holder.children) {
-        if (focus.cluster != holder.cluster) wrapText(context, holder);
+      if (holder.cluster != activeCluster || activeCluster == -1) {
```

Comment:
a little gross but I wanna make it so that if you mouse off of the clusters it shows everything thus the second part of the conditional